src/lib/draw-and-pick.js

```diff
@@ -206,11 +195,122 @@ export function pickLayers(gl, {
 }
 /* eslint-enable max-depth, max-statements */
 
+export function pickLayersByBoundingBox(gl, {
```

Comment:
See my concern in the RFC. This will not catch hidden objects which I think it expected in an area selection operation. But maybe I am missing the use case. So depending on what we decide, we should ask ourselves if it is worth including a possibly flawed picking function.